### ISSUE TITLE
Cannot Resolve

### ISSUE THREAD

**Issue by ejdelros08** [oxoooo/touch-image-view#1]
Hi,

I'm trying to include this library on my new project but gradle show me an error "cannot resolve" and peer no authenticated. It seems that gradle cannot find the source for this library. It may have been removed from where you've uploaded the library. I've previously used your library and I've not encountered any problem when it comes to including it on my project via Gradle. Can you help and check? 

Thanks!

**Comment by xingrz** [comment]
Hi @ejdelros08 

Did you include jitpack into your `repositories`?

``` gradle
repositories {
    jcenter()
    maven { url "https://jitpack.io" }
}
```

**Comment by ejdelros08** [comment]
Yes I have.

**Comment by xingrz** [comment]
Seems something wrong with jitpack. I've try to rebuild the library on jitpack but it doesn't work. I'll try to investigate the problem soon.

**Comment by xingrz** [comment]
or, you may try this:

``` gradle
compile 'ooo.oxo:touch-image-view:1.0.1'
```

**Comment by ejdelros08** [comment]
alright. Hoping to get this fixed asap, the project is urgent. Thank you very much for your response. Godbless

**Comment by ejdelros08** [comment]
Hi,

I've tried using 
`compile 'ooo.oxo:touch-image-view:1.0.1'`

but I'm still having the same issue. Do i need to remove the

`maven { url "https://jitpack.io" }`

Thanks

**Comment by xingrz** [comment]
No, don't remove that line.

Can you open the web site https://jitpack.io properly?

**Comment by ejdelros08** [comment]
Yes, I can view the website and tried syncing gradle again before sending this comment. Still the same problem.

**Comment by antoniolucasnobar** [comment]
I have the same error. On jipack site, I only found pullbacklayout. No image-view there.
